spec/integration/shared_examples/active_record_functions.rb

```diff
@@ -0,0 +1,41 @@
+RSpec.shared_examples 'Arproxy does not break the original ActiveRecord functionality' do
+  before do
+    # CREATE
+    ActiveRecord::Base.connection.create_table :products, force: true do |t|
+      t.string :name
+      t.integer :price
+    end
+    # INSERT
+    Product.create(name: 'apple', price: 100)
+    Product.create(name: 'banana', price: 200)
+    Product.create(name: 'orange', price: 300)
+  end
+
+  after(:all) do
+    ActiveRecord::Base.connection.drop_table :products
+  end
+
+  context 'SELECT' do
+    it { expect(Product.where(name: ['apple', 'orange']).sum(:price)).to eq(400) }
+  end
+
+  context 'UPDATE' do
+    it do
+      expect {
+        Product.where(name: 'banana').update_all(price: 1000)
+      }.to change {
+        Product.find_by(name: 'banana').price
```

Comment:
nit: Since the result can be `nil`,

```suggestion
        Product.find_by(name: 'banana')&.price
```

or simply using `find_by!`.

Reply:
https://github.com/cookpad/arproxy/pull/32/commits/fbf518bde021a60ae89a2378ba9050a8ad41d931